tools/summarize-topic

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import urllib.parse
```

Comment:
I think you might want to put this in `zulip/integrations/litellm/` with a `requirements.txt` in that directory; I think that's our intended mechanism for this sort of integration shipped with the package.

Reply:
Done.